api-gateway/src/test/java/com/petclinic/bffapigateway/domainclientlayer/VisitsServiceClientIntegrationTest.java

```diff
@@ -72,6 +81,43 @@ void getAllVisits() throws JsonProcessingException {
                 .expectNext(visitResponseDTO2)
                 .verifyComplete();
     }
+
+    @Test
+    void createVisitForPet_Valid() throws JsonProcessingException {
+        // Arrange
+        VisitRequestDTO visitRequestDTO = new VisitRequestDTO(
+                LocalDateTime.now(),
```

Comment:
We're going to have to fix this with the new date time formatter but we'll take care of it